modules/product/product.service.js

```diff
@@ -17,11 +17,19 @@ const { uploadProductImages } = require('./product.utils');
 
 class ProductsService {
```

Comment:
You need to populate all collections in all methods.

Reply:
fixed